apps/package.json

```diff
@@ -188,6 +188,7 @@
     "filesaver.js": "0.2.0",
     "jszip": "3.0.0",
     "karma-coveralls": "^1.1.2",
-    "query-string": "4.1.0"
+    "query-string": "4.1.0",
+    "sinon-chai": "^2.8.0"
```

Comment:
Should this be in `devDependencies`?  Or do we not care about that in a post-`yarn` world?